cidrgrep.go

```diff
@@ -0,0 +1,31 @@
+package cidrgrep
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"regexp"
+)
+
+// Filter filters input for lines containing an IP within cidr
+func Filter(input io.Reader, output io.Writer, cidr string) {
```

Comment:
```suggestion
func Filter(r io.Reader, w io.Writer, pattern string) {
```

`r` and `w` are conventional for readers and writers, and `pattern` is a little more obvious, isn't it?

Reply:
done